build.sbt

```diff
@@ -71,7 +71,7 @@ lazy val site = project.in(file("site"))
 // General Settings
 lazy val commonSettings = Seq(
   scalaVersion := "2.13.1",
-  crossScalaVersions := Seq(scalaVersion.value, "2.12.10"),
```

Comment:
Why disabling cross-compiling for `2.12.10`?

Reply:
maybe we could do it in a different `pr` because of `scala.collection.immutable.LazyList`

Reply:
Perfect then!